src/js/site/expandImage.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Add enlarge image capability for easier readability
+ */
+export function expandImageOnClick() {
+  // create references to the modal...
+  var modal = document.getElementById('expandModal');
```

Comment:
can we use `const` here?

Reply:
also, could we use `document.getElementByClass('modal')` instead? There doesnt seem to be another element with the class `modal` and the id `expandModal` has no css applied to it as well. 

you could still do `document.getElementById('expandModal')` but then I dont think we need the class, and we can change the css for modal to `#expandModal {}`?

Reply:
Great catch!

Reply:
it can't be const because i need to change the model with the close caption but it can be `let`

Reply:
gotcha, could we change the css for `.modal` to `#expandModal`? The modal class doesnt seem to be used anywhere else.

Reply:
Im not sure I follow for the close caption? I was able to test it out and `const` seems to be working well for modal 👍

Reply:
Can do the css changes! good point!